libraries/bezier-rs/src/subpath/transform.rs

```diff
@@ -509,6 +576,43 @@ mod tests {
 		subpath
 	}
 
+	#[test]
+	fn outline_with_single_point_segment() {
+		let subpath = Subpath::new(
+			vec![
+				ManipulatorGroup {
+					anchor: DVec2::new(20., 20.),
+					out_handle: Some(DVec2::new(10., 90.)),
+					in_handle: None,
+					id: EmptyId,
+				},
+				ManipulatorGroup {
+					anchor: DVec2::new(150., 40.),
+					out_handle: None,
+					in_handle: Some(DVec2::new(60., 40.)),
+					id: EmptyId,
+				},
+				ManipulatorGroup {
+					anchor: DVec2::new(150., 40.),
+					out_handle: Some(DVec2::new(40., 120.)),
+					in_handle: None,
+					id: EmptyId,
+				},
+				ManipulatorGroup {
+					anchor: DVec2::new(100., 100.),
+					out_handle: None,
+					in_handle: None,
+					id: EmptyId,
+				},
+			],
+			false,
+		);
+
+		let outline = subpath.outline(10., crate::Join::Round, crate::Cap::Round).0;
+		assert_eq!(outline.len(), 25);
```

Comment:
This feels like it could end up being somewhat flaky. Could you instead assert that none of the consecutive manipulator groups fall on top of one another?